x-pack/plugins/painless_lab/server/routes/api/execute.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { schema } from '@kbn/config-schema';
+
+import { RouteDependencies } from '../../types';
+import { API_BASE_PATH } from '../../../common/constants';
+import { isEsError } from '../../lib';
+
+const bodySchema = schema.object({
+  script: schema.object({
+    source: schema.string(),
```

Comment:
We'll need to update this schema to support the other parameters that will be provided if the user specifies "Parameters" or the other options provided by the other "Context" choices. For example, if you specify "Parameters" as in the screenshot below the request fails because it's invalidated by this schema:

![image](https://user-images.githubusercontent.com/1238659/76475229-6f908d80-63bb-11ea-9471-35e5df99b015.png)